examples/apache-maven/custom-pipelines-hello-world/src/main/java/co/decodable/examples/cpdemo/DataStreamJob.java

```diff
@@ -35,24 +35,21 @@ public class DataStreamJob {
 	public static void main(String[] args) throws Exception {
 		StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
 
-		DecodableStreamSource<String> source =
-				DecodableStreamSource.<String>builder()
```

Comment:
What's the reason for changing how this API is bootstrapped? The static method makes for easy discoverability on the public `DecodableStreamSource` type, and we should retain that.

Reply:
changed https://github.com/decodableco/decodable-pipeline-sdk/pull/41/commits/fa4608a93d960b19f881afb13f41a906090d0d3f